app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    def wrapper(*args) -> str:
```

Comment:
`wrapper` does not always return `str`, it could return `Any`. Indicate this in the return type annotation.